app/controllers/admin/institutions_controller.rb

```diff
@@ -1,4 +1,16 @@
 module Admin
   class InstitutionsController < Admin::ApplicationController
+    def import_receivers
+      institution = Institution.find(params[:id])
+      csv_file = params["file"].tempfile
```

Comment:
Why is this a string key instead of a symbol?

Reply:
Also, will this work with Heroku's temporary filesystem? I don't know what restrictions they currently place.